### DISCUSSION TITLE
How can I add a button or an icon for fullscreen mode on mobile devices?

### DISCUSSION THREAD

**Discussion by nikomaxia** [hakimel/reveal.js#3146] [Q&A] [answered]
At a pc or mac if you press F button it goes on fullscreen.
On mobile devices I want to add an icon on the top right of the screen so if you tap on it this will make the presentation fullscreen.

How can I find the code for F button to embbed it on the icon?

Thanks!

**Comment by Martinomagnifico** [comment, marked as answer]
Try this: https://github.com/Martinomagnifico/reveal.js-fsfx
